bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -103,15 +103,16 @@ metadata:
     categories: AI/Machine Learning, Big Data
     certified: "False"
     containerImage: quay.io/opendatahub/opendatahub-operator:v2.19.0
-    createdAt: "2024-11-21T09:22:44Z"
+    createdAt: "2024-12-02T18:57:19Z"
     olm.skipRange: '>=1.0.0 <2.19.0'
     operators.operatorframework.io/builder: operator-sdk-v1.31.0
     operators.operatorframework.io/internal-objects: '["featuretrackers.features.opendatahub.io",
       "codeflares.components.opendatahub.io", "dashboards.components.opendatahub.io",
       "datasciencepipelines.components.opendatahub.io", "kserves.components.opendatahub.io",
       "kueues.components.opendatahub.io", "modelmeshservings.components.opendatahub.io",
       "modelregistries.components.opendatahub.io", "rays.components.opendatahub.io",
-      "trainingoperators.components.opendatahub.io", "trustyais.components.opendatahub.io",  "workbenches.components.opendatahub.io"]'
+      "trainingoperators.components.opendatahub.io", "trustyais.components.opendatahub.io",  "workbenches.components.opendatahub.io",
+      "monitorings.services.opendatahub.io" ]'
```

Comment:
is it part of #1389?

Reply:
I think so, the descriptions mentions: _Should be merged after https://github.com/opendatahub-io/opendatahub-operator/pull/1389_

Reply:
> I think so, the descriptions mentions: _Should be merged after #1389_

it says, but it is a part of the patch 3af04a240890 ("Update api version for internal apis to v1alpha1")

Reply:
we should aim to get the monitoring PR in first,  this one included the other, will need a big rebase eventually.

Reply:
IIUC it's supposed to be one patch on top on the #1389 So, not such a big rebase